src/js/components/Button/doc.js

```diff
@@ -259,6 +259,21 @@ export const themeDoc = {
     type: 'string',
     defaultValue: '8px',
   },
+  'button.transition.timing': {
+    description: 'The transition timing function',
+    type: 'string',
+    defaultValue: 'ease-in-out',
+  },
```

Comment:
Can we organize the styles with alphabetical order and move this block after the `button.transition.properties` block?